integration-tests/pom.xml

```diff
@@ -95,33 +95,29 @@
 							<workingDirectory>${sidecar.dir}</workingDirectory>
 						</configuration>
 					</execution>
+				</executions>
+			</plugin>
 
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>exec-maven-plugin</artifactId>
+				<version>3.2.0</version>
+				<executions>
 					<execution>
-						<id>start-sidecar</id>
-						<goals>
-							<goal>npx</goal>
-						</goals>
 						<phase>pre-integration-test</phase>
-						<configuration>
-							<arguments>pm2 -l sidecar.log --time start --name=sidecar npm -- run start</arguments>
-							<skip>${skipTests}</skip>
```

Comment:
Could we retain the skip configuration, so that MTX Sidecar is not started, in case tests are skipped? Or would this anyway be the case, because phase `pre-integration-test` is never executed?